src/intents.js

```diff
@@ -46,6 +46,14 @@ function injectService (url, element, intent, data) {
         return event.source.postMessage(data, event.origin)
       }
 
+      if (handshaken && event.data.type === `intent-${intent._id}:size`) {
+        ['width', 'height', 'maxWidth', 'maxHeight', 'minWidth', 'minHeight'].forEach(prop => {
+          if (event.data.document[prop]) element.style[prop] = event.data.document[prop]
```

Comment:
Ok now I understand what is done here and why CSS values are directly transmitted. Again, I am not a big fan of letting the service do whatever he wants with client's iframe CSS values. It should limit its role to transmit width and height. Nothing more is needed. Again, if @CPatchane and @y-lohse agree with this, let's keep it.

Summary of what I suggest: 
```js
['width', 'height'].forEach(property => {
  element.style[property] = `${event.data.dimensions[property]}px`  || element.style[property]
}
```
It is reeeeaaaally difficult for me to consider passing directly a CSS value as a string containing `px`, sorry about that.